sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -1601,6 +1602,11 @@ def test_pivot_no_index_provided_on_multiindex(self):
 
 class GroupByTest(_AbstractFrameTest):
   """Tests for DataFrame/Series GroupBy operations."""
+  @staticmethod
+  def median_sum_fn(x):
+    warnings.filterwarnings("ignore", message="Mean of empty slice")
+    return (x.foo + x.bar).median()
```

Comment:
Could we use a [context](https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings) here to make sure the filters get reset?

Reply:
Thanks for the suggestion. Applied.